src/test/java/net/snowflake/ingest/streaming/internal/StreamingIngestIT.java

```diff
@@ -462,7 +467,7 @@ public void testMultiColumnIngest() throws Exception {
 
     // Open a streaming ingest channel from the given client
     SnowflakeStreamingIngestChannel channel1 = client.openChannel(request1);
-    long timestamp = System.currentTimeMillis();
+    long timestamp = Instant.now().toEpochMilli();
```

Comment:
Out of curiosity, why this change?

Reply:
I thought we want everything to be UTC, `System.currentTimeMillis` will be depended on the timezone, right?

Reply:
https://docs.oracle.com/javase/7/docs/api/java/lang/System.html#currentTimeMillis() Will give you epoch millis, so not timezone dependent

Reply:
I see, I think both of them are doing the same thing, revert it